src/cliargs.lua

```diff
@@ -520,11 +520,14 @@ function cli:print_usage(noprint)
   -- print the USAGE heading
   local msg = "Usage: " .. tostring(self.name)
   if self.optional[1] then
-    msg = msg .. " [OPTIONS] "
+    msg = msg .. " [OPTIONS]"
+  end
+  if self.required[1] or self.optargument.maxcount > 0 then
```

Comment:
just curious, is there a reason we're not checking `#self.required` instead of `self.required[1]` ?